cczoo/horizontal_fl/recommendation_system/Makefile

```diff
@@ -18,6 +18,8 @@
 ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
 # ENTRYPOINT ?= $(realpath $(shell sh -c "command -v python3"))
 
+RA_TYPE ?= none
```

Comment:
Should we set the default value with "dcap"?

Reply:
OK, will modify it.